internal/uvm/create_wcow.go

```diff
@@ -280,7 +280,7 @@ func CreateWCOW(ctx context.Context, opts *OptionsWCOW) (_ *UtilityVM, err error
 			},
 		}
 
-		uvm.scsiLocations[0][0] = newSCSIMount(uvm, doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path, "", "", 1, 0, 0, false)
+		uvm.scsiLocations[0][0] = newSCSIMount(uvm, doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path, "", doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Type_, "", 1, 0, 0, false)
```

Comment:
Maybe let this function call span multiple lines now, it's getting pretty long hahah

```go
uvm.scsiLocations[0][0] = newSCSIMount(
                                    uvm, 
                                    doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path, 
                                    "", 
                                    doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Type_, "",  
                                    1, 
                                    0, 
                                    0, 
                                    false,
                            )

Reply:
Ok github butchered that, but you get the idea